Trimble.Modus.Components/Controls/TMSwitch/TMSwitch.xaml.cs

```diff
@@ -0,0 +1,210 @@
+using System.Windows.Input;
+using Trimble.Modus.Components.Constant;
+using Trimble.Modus.Components.Enums;
+using Trimble.Modus.Components.Helpers;
+
+namespace Trimble.Modus.Components;
+
+public partial class TMSwitch : ContentView
+{
+    protected EventHandler<TMSwitchEventArgs> _clicked;
+    private bool _switchSelected;
+    private TMSwitchEventArgs switchEventTrue, switchEventFalse;
```

Comment:
No need to store them in variables. Whenever you need them, create them new `new TMSwitchEventArgs(true)`. Because if the user changes the value in the demo app at runtime, that would create problems

Reply:
Changed